src/test/ui/parser/issue-88276-unary-plus.rs

```diff
@@ -0,0 +1,13 @@
+// run-rustfix
+#[allow(unused_parens)]
+fn main() {
+    let _ = +1; //~ ERROR leading `+` is not supported
+    let _ = -+(1+2)*3; //~ ERROR leading `+` is not supported
+    let _ = -+-+(1+2)*3; //~ ERROR leading `+` is not supported
+                         //~| ERROR leading `+` is not supported
```

Comment:
This is an interesting case, because with the suggestions applied, this looks like C's prefix decrement `--foo`, which we don't have. We should have a lint against it 😅

Reply:
That's a good idea for a lint!

Reply:
Filed https://github.com/rust-lang/rust/issues/88669 so we don't forget.